envi/archs/arm/disasm.py

```diff
@@ -5,36 +5,19 @@
 import envi
 import envi.bits as e_bits
 
-from envi.bits import binary
-
-#import sys
-#import struct
-#import traceback
-
-#import envi
-#import envi.bits as e_bits
-#from envi.bits import binary
-
 from envi.archs.arm.const import *
 from envi.archs.arm.regs import *
 
-# FIXME: TODO
-# FIXME ldm sp, { pc } seems to not get marked NOFALL
-# FIXME ldm sp, { pc } should probably be marked IF_RET too...
-# FIXME b lr / bx lr should be marked IF_RET as well!
-# FIXME encoding for swi instruction ( <4 cond> 1111 <24 bytes immediate> ) is totally horked (it's in p_uncond)
-# FIXME some arm opcode values are ENC << and some are ENC and some are etc..
-#       (make all be ENC_FOO << 16 + <their index>
-
-# FIXME the following things dont decode correctly
-# 5346544e    cmppl   r6, #1308622848
+# FIXME:   codeflow needs to identify the following pattern as a call with fallthrough
```

Comment:
it can be an ARM-specific fixup pass. not for this PR, although i can remove this comment.

Reply:
ideally we would be able do it right at the decoding, since we so often parse the opcode and grab iflags from there... instead of grabbing the location entry flags.  problem is the decoder doesn't track state (nor should it)

we can update the location based on an analysis module, but then we'd have to be certain to use the location for anything that looks for calls/jmps/fallthrough.

we clearly can do something in getBranches().

ugh.